examples/mugen/retrieval/README.md

```diff
@@ -0,0 +1,41 @@
+# MUGEN Retrieval
+
+This directory contains model components for MUGEN's video-text retrieval model, a tutorial notebook for the model usage (better viewed on [nbviewer](https://nbviewer.org/github/facebookresearch/multimodal/blob/main/examples/mugen/retrieval/evaluation.ipynb)), and training and evaluation scripts.
+
+## Model
+MUGEN's video-text retrieval model follows from [VideoCLIP](https://arxiv.org/abs/2109.14084), a contrastive model for video and text.
+
+The name "VideoCLIP" refers to its similarities to OpenAI's [CLIP](https://arxiv.org/abs/2103.00020), which was originally proposed for zero-shot learning of image classification tasks by “drawing cues” from text data with the corresponding visual concepts. Unlike various predecessor models based on supervised learning, CLIP does not have to be trained on the task-specific datasets or fine-tuned with a task-specific head. The model learns a joint embedding space for both image and text data and optimizes a scaled cosine similarity function between the image and text embedding vectors. The loss function is the sum of the normalized cosine similarities for every pair of image-and-text samples. Each embedding is trained with a modality specific transformer encoder, e.g., a multi-head transformer for text, vision transformer (ViT) for image.
```

Comment:
>e.g., a multi-head transformer for text, vision transformer (ViT) for image.

This may appear to be ambiguous for readers with no experience about CLIP:
- For the image encoder, both ViT and ResNet are used.
- For text, I think "transformer" is enough to be succinct.

Reply:
changed to `modality specific encoder, e.g., a transformer for text, vision transformer (ViT) or ResNet for image.`